src/components/tester/TesterHome.jsx

```diff
@@ -1,61 +1,27 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import MapExplorer from '../../containers/maps/MapExplorer';
 import { connect } from 'react-redux';
-import { Actions } from 'jumpstate';
-import {
-  CLASSROOMS_STATUS, CLASSROOMS_INITIAL_STATE, CLASSROOMS_PROPTYPES,
-} from '../../ducks/classrooms';
+
+const mapConfig = require('../../lib/wildcam-darien.mapConfig.json');
```

Comment:
Webpack 3 has json-loader support built in. Did you have issues using new ES6 module import syntax for this? Just curious why you're using CommonJS require here.